modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -1078,4 +1109,53 @@ public String getRoleV2ResourceId(String roleName, String audienceType, String O
 		}
 		return null;
 	}
+
+	public String createDomainAPIs(DomainAPICreationModel domainAPICreationModel) {
```

Comment:
format this particular method so it adds a line

Reply:
Updated with 4412a6451aab9f4e6c947538138d3c80ad3f36b7